static/js/services/add-attachment.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Service to create and attach data to a couchdb doc
+ */
+angular.module('inboxServices').factory('AddAttachment',
+  function() {
+    'ngInject';
+    'use strict';
+
+    return function(doc, name, content, contentType) {
+      if (!doc._attachments) {
+        doc._attachments = {};
+      }
+      var blob = new Blob([ content ], { type: contentType });
```

Comment:
Any idea if `Blob` actually does anything with the `type` option?  Is it returned later when you retrieve it?  Can't see anything in the docs.